src/Plugin/EntityBrowser/Display/Modal.php

```diff
@@ -177,7 +173,7 @@ public function addAjax(array &$form) {
     );
 
     $form['actions']['submit']['#ajax'] = array(
-      'callback' => array(get_class($this), 'widgetAjaxCallback'),
+      'callback' => array($this, 'widgetAjaxCallback'),
```

Comment:
Yeah, I did see that causing problems, for for some reason with this new approach, it doesn't break it for me.  We do need to access properties on $this, so maybe we should look again at why adding $this is wasn't working?? I will have time later in the week to take a look, but I think its important that we are able to add an ajax callback that can access properties on the object.